docs_samples/01_python_app_insights_overview.md

```diff
@@ -0,0 +1,239 @@
+# Getting Stared with App Insights for Python
+
+This procedure configures your Python application to send telemtery tot he Application Insights feature of the Azure Monitor service.  It works with any Python application on-premises or in the cloud.
+
+## Prerequisites
+
+To work with App Insights for Python, you need to:
+
+- Install Python (version 3.9.x).
```

Comment:
Do these examples only work on v3.9.x?

Reply:
They should run on 3.7.x or higher.  I can redo all the steps with 3.7 to verify if you think its worth it to do so?

Reply:
No need, I would just add an excerpt stating that they have been CONFIRMED to work with Python 3.9 and not restrict the user to a certain version.